launch/sim.launch.py

```diff
@@ -11,32 +11,27 @@
     SetEnvironmentVariable,
     Shutdown,
 )
-from launch.conditions import IfCondition
 from launch.launch_description_sources import PythonLaunchDescriptionSource
-from launch.substitutions import LaunchConfiguration, PythonExpression
+from launch.substitutions import LaunchConfiguration
 
 
 def generate_launch_description():
     bringup_dir = Path(get_package_share_directory("rj_robocup"))
     launch_dir = bringup_dir / "launch"
 
+    config_yaml = LaunchConfiguration("config_yaml", default="sim.yaml")  # noqa: F841
+    use_internal_ref = LaunchConfiguration(  # noqa: F841
```

Comment:
tested the following:
`ros2 launch rj_robocup sim.launch.py use_internal_ref:=False`

and launched ssl-game-controller as usual, but this did not work for me. Switching False to True works fine so I know it's not a syntax thing.

Reply:
Should be fixed now